### DISCUSSION TITLE
Welcome to ping-viewer Discussions!

### DISCUSSION THREAD

**Discussion by patrickelectric** [bluerobotics/ping-viewer#941] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by constancekoh1** [comment]
Hello, why isn't Ping Viewer detecting any information? 
I am seeing this under default: 
'' default[Warning]: PingGroupBox.qml(6) qrc:/PingGroupBox.qml:6:5: QML Binding: Not restoring previous value because restoreMode has not been set.
This behavior is deprecated.
You have to import QtQml 2.15 after any QtQuick imports and set
the restoreMode of the binding to fix this warning.
In Qt < 6.0 the default is Binding.RestoreBinding.
In Qt >= 6.0 the default is Binding.RestoreBindingOrValue.''

**Comment by patrickelectric** [comment]
@constancekoh1 can you explain what you mean by:
 
 < Hello, why isn't Ping Viewer detecting any information?
